sympy/solvers/__init__.py

```diff
@@ -32,3 +32,7 @@
 from .decompogen import decompogen
 
 from .solveset import solveset, linsolve, linear_eq_to_matrix
+
+from ..core.singleton import S
+Complexes = S.Complexes
```

Comment:
It may be a mess to fix some of these issues. If you can't figure them out, just add imports for whatever fancy sets you can.